html5/js/Client.js

```diff
@@ -79,7 +79,7 @@ class XpraClient {
       "rgb32",
       "rgb24",
       "scroll",
-      "void",
+      "void"
```

Comment:
Is it because the linter is complaining? If so, can we silence it?
(it's much better ***with*** the trailing comma because then you can add to the list without causing a diff on an unrelated line above)

Reply:
force of habit :) fixed.